### PR TITLE
LPE fix to enable visual odom. only navigation.

### PR DESCRIPTION
**Pull request by jgoppert**

### CODE REVIEW

**Comment by jgoppert** [comment]
Due to the restrictions on eph for position mode even if the velocity was good due to visual odometry I couldn't do a position hold since my x y covariance was growing since no position estimates were available.

This patch circumvents this by using xyvalid to set the eph and epv under the triggers within commander for good global and local positoin. This makes me think that commander should just let the estimator decide when xy is valid, or some special condition needs to be added to allow odometry only navigation relative to a known starting position.

**Comment by ecmnet** [comment]
@jgoppert You hurry fast with LPE - hard to follow :-). I just tried to use https://github.com/PX4/Firmware/commit/7c2798269cc360ab4a39532fd2cc351fdf141faa with Release 1.4.1 RC1 (as this is my current working version) and find vision not recognized at all. Applying this patch did not help for me. I can see LPE vision being initialized, but XY is not published (although z is). Neither in LPOS nor in GPOS. Any idea what is going on?
I also saw vision timeout set to 10secs. For me (as vision is the only source of position indoors) this seems to far to long for position recovery.

**Comment by jgoppert** [comment]
@ecmnet  Have you had a chance to try this out? https://github.com/PX4/Firmware/pull/5574

I just rebased it on master so should be good to test.

**Comment by ecmnet** [comment]
@jgoppert Not yet, will try asap.
